docs/content/users/basics/developer-tools.md

```diff
@@ -75,6 +75,8 @@ MailHog will **not** intercept emails if your application is configured to use S
 
 If you’re using SMTP for outgoing mail—with [Symfony Mailer](https://www.drupal.org/project/symfony_mailer) or [SMTP](https://www.drupal.org/project/smtp) modules, for example—update your application’s SMTP server configuration to use `localhost` and MailHog’s port `1025`.
 
+For Drupal 9 `settings.ddev.php` overrides the Symfony Mailer sendmail configuration to use MailHog.
```

Comment:
```suggestion
For Drupal 9+ `settings.ddev.php` overrides the Symfony Mailer sendmail configuration to use MailHog.
```

Reply:
There's still no D10 release for drupal/symfony_mailer. That's why I did it only for D9.

Reply:
OK thanks! You're on top of *everything*. Although it wouldn't do any harm to add it to d10 would it? These are a no-op of people don't have symfony mailer installed?